ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -485,5 +485,4 @@ withServerNode muxTracer handshakeTracer errorPolicyTracer tbl stVar addr encode
 
       acceptException :: Socket.SockAddr -> SomeException -> IO ()
       acceptException a e = do
-        traceWith (WithAddr a `contramap` errorPolicyTracer) $ ErrorPolicyAccept e
-        throwIO e
+        traceWith (WithAddr a `contramap` errorPolicyTracer) $ ErrorPolicyAcceptException e
```

Comment:
Must rethrow async exceptions, or at the very least, `AsyncCancelled` and `ThreadKilled` exceptions.

Reply:
This is the exception handler around `Network.Socket.accept` right? I'd recommend only catching `IOExceptions` here.

Reply:
Yes, we should only catch `IOException`s here.